README.md

```diff
@@ -27,3 +27,22 @@ All [configuration variables](https://github.com/eccenca/gui-elements/blob/devel
     * `$eccgui-size-typo-base-lineheight`: only ratio to font size, no unit!
     * `$eccgui-size-type-levelratio`: ratio without unit! used to calculate different text sizes based on `$eccgui-size-typo-base`
     * `$eccgui-size-block-whitespace`: white space between block level elements, currently only `px` is supported
+
+## Running Storybook
+
+All stories are in the respective components, extensions and cmem folders. 
+All stories have the extension `*.stories.tsx`
+To get started running story book, run in terminal `yarn storybook`. 
+
+### Adding and running story tests. 
+
+* To add tests to a story:
```

Comment:
This is correct but may miss some information, for file name patterns like `Button.stories.js` and `Button.test.tsx` it is configured to match this auomatically, in case the file names cannot be matched, the jest test files need to be attached in the story directly.